### DISCUSSION TITLE
MultiVI and Scenic+ cross-compatibility

### DISCUSSION THREAD

**Discussion by benduc** [aertslab/scenicplus#134] [General]
Dear Scenic+ developers,
I recently found that [MultiVI](https://www.biorxiv.org/content/10.1101/2021.08.20.457057v2) is a useful tool for batch correction of multiomic datasets. It uses "a deep generative model for probabilistic and integrative analysis of scRNA-seq, scATAC-seq and paired multi-modal data". I have a question about its compatibility with Scenic+:

- **Can one use MultiVI for batch correction of multiomic datasets (10X Multiome for example) and run the batch-corrected output in Scenic+?**
- If not, do you yourself recommend batch correction methods for multiomic datasets?

Thank you for all your work!

**Comment by SeppeDeWinter** [comment]
Hi @benduc 

We usually don't do any batch effect correction before running SCENIC+

Best,

Seppe
